SECURITY.md

```diff
@@ -14,6 +14,19 @@ TorchServe as much as possible relies on automated tools to do security scanning
 2. Docker Scanning: Using Snyk
 3. Code Analysis: Using CodeQL
 
+## Important Security Guidelines
+
+1. TorchServe listens on the following ports
+    1. HTTP - `8080`, `8081`, `8082`
+    2. gRPC - `7070`, `7071`
+
+    These ports are accessible to `localhost` by default.  The address can be configured by following the [guide](https://github.com/pytorch/serve/blob/master/docs/configuration.md#configure-torchserve-listening-address-and-port)
+    TorchServe does not prevent users from configuring the address to be `0.0.0.0`. Please be aware of the security risks if you use `0.0.0.0`
+2. TorchServe's Docker image is conigured to listen to `localhost` by [default](https://github.com/pytorch/serve/blob/master/docker/config.properties)
+
```

Comment:
also mention that torchserve can run arbitrary python files so don't download mar files you don't trust from the internet

Reply:
done

Reply:
the updated language is good but please explicitly also mention that torchserve executes arbitrary python files when running a mar file, the risk is not just in downloading

Reply:
Got it. So, how do they verify it? We should mention that too

Reply:
There is no generic way to verify if a piece of python code is safe, they key is that it needs to be code from a source they trust